app/services/activitypub/process_account_service.rb

```diff
@@ -127,12 +129,14 @@ def set_fetchable_attributes!
     begin
       @account.avatar_remote_url = image_url('icon') || '' unless skip_download?
       @account.avatar = nil if @account.avatar_remote_url.blank?
+      @account.avatar_description = @json['avatarDescription'] || '' unless @account.avatar_remote_url.blank?
     rescue Mastodon::UnexpectedResponseError, *Mastodon::HTTP_CONNECTION_ERRORS
       RedownloadAvatarWorker.perform_in(rand(30..600).seconds, @account.id)
     end
     begin
       @account.header_remote_url = image_url('image') || '' unless skip_download?
       @account.header = nil if @account.header_remote_url.blank?
+      @account.header_description = @json['headerDescription'] || '' unless @account.header_remote_url.blank?
```

Comment:
If header isn't present, then both can be nil (or maybe description as an empty string) — there's no point storing the description if there is actually no image to describe